### ISSUE TITLE
the number of base classifiers

### ISSUE THREAD

**Issue by GouhaoC** [graphNN/GEN1#1]

**Comment by graphNN** [comment]
Thank you for your comment. Tables 5 and 6 are mainly to verify the overfitting of GEN, and the complexity, where a lot of C is omitted, leading to the fact that it looks like GEN performs best when C=1. In fact, Table 2 is the optimal hyperparameters provided in this paper, please refer to Table 2 when you need to reproduce the related experiments.

**Comment by graphNN** [comment]
> In the experiments regarding the number of base classifiers, Tables 5 and 6 show that when (C=1), the GEN model performs best on the training set, validation set, and test set. What is the purpose of ensembling then?

Thank you for your comment. Tables 5 and 6 are mainly to verify the overfitting of GEN, and the complexity, where a lot of C is omitted, leading to the fact that it looks like GEN performs best when C=1. In fact, Table 2 is the optimal hyperparameters provided in this paper, please refer to Table 2 when you need to reproduce the related experiments.

**Comment by GouhaoC** [comment]
Thank you for your answer. There are some problems when I run the code. Is there anything that needs special modification?

**Comment by graphNN** [comment]
> Thank you for your answer. There are some problems when I run the code. Is there anything that needs special modification?

Please let me know the exact problem via email (duan@gzhu.edu.cn), thanks!
